tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -342,25 +342,24 @@ def make_csv_dataset_v2(
   feature data, and labels is a `Tensor` containing the batch's label data.
 
   Usage Example:
-
   Using IRIS dataset to show how to convert .csv file into a dataset.
-  
+
   ```python
-  >> train_dataset_url = "https://storage.googleapis.com/download.tensorflow.org/data/iris_training.csv"
-  >> train_dataset_fp = tf.keras.utils.get_file(fname=os.path.basename(train_dataset_url), origin=train_dataset_url)
-  >> column_names = ['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'species']
-  >> feature_names = column_names[:-1]
-  >> label_name = column_names[-1]
-  >> batch_size = 32
-  >> train_dataset = tf.data.experimental.make_csv_dataset(
-      train_dataset_fp,
-      batch_size,
-      column_names=column_names,
-      label_name=label_name,
-      num_epochs=1)
-  >> features_batch, labels_batch = next(iter(train_dataset))
-  >> print(features_batch)
-  <OrderedDict([(..., <tf.Tensor: shape=(32,), dtype=float32, numpy=array([...], dtype=float32)>)])>
+  train_dataset_url = "https://storage.googleapis.com/download.tensorflow.org/data/iris_training.csv"
+  train_dataset_fp = tf.keras.utils.get_file(fname=os.path.basename(train_dataset_url), origin=train_dataset_url)
+  column_names = ['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'species']
+  feature_names = column_names[:-1]
+  label_name = column_names[-1]
+  batch_size = 32
+  train_dataset = tf.data.experimental.make_csv_dataset(
+    train_dataset_fp,
+    batch_size,
+    column_names=column_names,
+    label_name=label_name,
+    num_epochs=1)
+  features_batch, labels_batch = next(iter(train_dataset))
+  print(features_batch)
+  >> <OrderedDict([(..., <tf.Tensor: shape=(32,), dtype=float32, numpy=array([...], dtype=float32)>)])>
```

Comment:
Why this change?

Reply:
For consistency with the rest of the file.

Reply:
We would rather have https://github.com/tensorflow/tensorflow/blob/fda0768976e19aec6f7cb7b0c0a54ec160243a93/tensorflow/python/data/experimental/ops/readers.py#L663-L669 and other similar examples (although this one is the only one that has `>>` on the output) rewritten to the doctest format.

Plus, this doesn't solve the sanity pylint failure at all.